test/consts/gateway.go

```diff
@@ -1,5 +1,5 @@
 package consts
 
 const (
-	GatewayCRDsKustomizeURL = "github.com/kubernetes-sigs/gateway-api/config/crd?ref=71fee1c2808fa19a5f19d952d155fc072cf9324c"
+	GatewayCRDsKustomizeURL = "github.com/kubernetes-sigs/gateway-api/config/crd/experimental?ref=v0.5.0"
```

Comment:
Previously we had been (I think) using stable. Did we indeed want to move to and develop against experimental?